### ISSUE TITLE
Missing api for create nat port foward

### ISSUE THREAD

**Issue by lechuhuuha** [sjafferali/pfsense-api-goclient#60]
Hi i see that in this api is missing api for control nat port forward

![image](https://github.com/sjafferali/pfsense-api-goclient/assets/59781669/d128af4a-ea5d-4269-84a8-749b61d5070c)

do you need a pr or am i missing something here?

Thanks

**Comment by sjafferali** [comment]
@lechuhuuha Yeah, those methods have not been implemented in this golang client yet. I have been a bit pre-occupied and havent been able to complete the changes needed for the upcoming version 2 of the pfsense-api package.

I'll try to get this added as soon as I can, but feel free to submit a PR if you end up having the bandwidth to add support for this endpoint before that. Thanks for the issue!
